tests/bam/bam_pb.robot

```diff
@@ -601,7 +601,7 @@ BEPB_DIMENSION_KPI_EVENT
 
 
 BEPB_KPI_STATUS
-	[Documentation]	bbdo_version 3 use kpi_status message.
+	[Documentation]	bbdo_version 3 use kpi_status, Create a new BA for a specific service, start the broker and engine,check the status of a service three times until hard and set it to critical,verifing that the status of the service has been updated in the database and the timestamp of the last status change is correct.
```

Comment:
the 'three times' is too much in the documentation, the goal as you say, is that the service is in HARD state.
And then, since the service is in hard state, it must impact the kpi. Try to make something shorter please.